docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1647 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    gdf
</pre><p>gdf is a somewhat impenetrable name (if the pandas dataframe is called <code>pdf</code> why is the cudf dataframe not called <code>cdf</code> ?). If this is didactic, perhaps use <code>pandas_df</code> and <code>cudf_df</code> throughout?    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>